serviceWrappers/linux/kLiveController

```diff
@@ -57,7 +57,7 @@ loadNvm() {
 
 start() {
     [ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
-    loadNvm
+    #loadNvm
```

Comment:
Question is why. Like I wrote, NodeJS provides perfectly usable RPM and deb packages in their official repos. See:
https://nodejs.org/en/download/package-manager/